packages/shared/src/dto/step-schemas/generate-preview-response.dto.ts

```diff
@@ -59,8 +59,14 @@ export class ControlPreviewIssue {
   variableName?: string;
   message: string;
 }
+export class MasterPayload {
```

Comment:
what do you think about changing the name here to something else?

```suggestion
export class MainPayload {
```

Reply:
I agree. The name is too generic and introduces a master payload concept that doesn't exist in the product. As said earlier, we should eliminate it https://github.com/novuhq/novu/pull/6772/files#r1827360737

Reply:
is main payload exists?

Reply:
This is just a grouping that we don't have a strong argument for yet. I suggest we use separate parameters for payload, subscriber, and steps as they all have a very specific meaning in Novu land.

If we need a new umbrella term, we will add it in the future in a more structured way.